client/src/js/agreement/CreateAgreementContainer.jsx

```diff
@@ -28,9 +25,45 @@ function CreateAgreementContainer () {
   const classes = useStyles()
   return (
     <Paper elevation={23} className={classes.root}>
-      <h2 className={classes.h2}>Sign a CLA</h2>
-      <p className={classes.textCenter}>Who will you be submitting contributions on behalf of?</p>
       <Grid container spacing={3} justify='center'>
+        <Grid item xs={12}>
+          <h2 className={classes.h2}>Sign a CLA</h2>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2" className={classes.textCenter}>
+                When to sign an Individual CLA?
+              </Typography>
+              <Typography variant="body2" component="div">
+                I should sign this CLA if:
+                <ul>
+                  <li>I'm making contributions
+                    on my own free time and no-one is contracting me for them.</li>
```

Comment:
I'm making contributions in my free time with my own resources. I am not being paid to contribute.

Reply:
I'm making contributions of code I've created in my free time (not under contract), and I hold the copyright on my contributions.